packages/xsnap/src/avaXS.js

```diff
@@ -151,24 +155,93 @@ async function runTestScript(
 }
 
 /**
- * @param {string[]} argv
+ * Get ava / ava-xs config from package.json
+ *
+ * @param { string[] } args
+ * @param {{ packageFilename?: string }} opts
```

Comment:
Avoiding `null` implies `packageName: string | void` over `?` (which is `| null`) or
```suggestion
 * @param {Object} options
 * @param {string} [options.packageFilename]
```